exercises/D-logical-operators/exercise.js

```diff
@@ -11,14 +11,14 @@ var cssLevel = 4;
 
 // Finish the statement to check whether HTML, CSS knowledge are above 5
 // (hint: use the comparison operator from before)
-var htmlLevelAbove5;
-var cssLevelAbove5;
+var htmlLevelAbove5 = htmlLevel > 5;
+var cssLevelAbove5 = cssLevel > 5;
 
 // Finish the next two statement
 // Use the previous variables and logical operators
 // Do not "hardcode" the answers
-var cssAndHtmlAbove5;
-var cssOrHtmlAbove5;
+var cssAndHtmlAbove5 = htmlLevelAbove5 && cssLevelAbove5 > 5;
+var cssOrHtmlAbove5 = htmlLevelAbove5 || cssLevelAbove5 > 5;
 
```

Comment:
plz. test the value and type of the cssLevelAbove5 > 5, cssLevelAbove5 is already a boolean, try to test with different value of cssLevel and you will see why it's wrong